lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -441,7 +457,8 @@ class TextEditingElement {
       _positionInputElementTimer?.cancel();
       _positionInputElementTimer = Timer(_delayBeforePositioning, () {
         _canPosition = true;
-        _positionElement();
+        // Position element.
+        _geometricInfo?.applyToDomElement(domElement);
```

Comment:
Is there a reason we are not calling `positionElement()` here? Might be worth explaining in a comment.

Reply:
positionElement method almost does the same operation.

done. changed.

Reply:
done.